DALI.tex

```diff
@@ -1020,11 +1020,12 @@ \subsection{JSON}
 (e.g., as a \xmlel{GROUP} for a \xmlel{PARAM} or as unrolled
 \xmlel{FIELD}\/s or a separate table for table cells) is preferred.
 
-\xmlel{PARAM}\/s and \xmlel{FIELD}\/s containing JSON values should be
-marked up with a \xmlel{datatype} of \verb|char| or \verb|unicodeChar|,
-with variable or fixed \xmlel{arraysize} and an \xmlel{xtype} of
-\verb|json|.
+JSON values serialised in VOTable or described in service parameters should have
+the following VOTable type metadata:
 
+\begin{verbatim}
+datatype="char" arraysize="*" xtype="uri"
```

Comment:
That should be xtype="json"

Reply:
yes, that was just a cut and paste of the style I found worked best